src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/Setup/useDefaultColumnConfig.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import { useMemo } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import {
+    dataElementTypes,
+    type TrackerProgram,
+    type DataElement,
+} from '../../../../../metaData';
+import type {
+    MainColumnConfig,
+    MetadataColumnConfig,
+    TeiWorkingListsColumnConfigs,
+} from '../types';
+
+const getDefaultMainConfig = (): Array<MainColumnConfig> => [{
+    id: 'regUnit',
+    visible: false,
+    type: dataElementTypes.TEXT,
+    header: i18n.t('Registering unit'),
+}, {
+    id: 'regDate',
+    visible: false,
+    type: dataElementTypes.DATE,
+    header: i18n.t('Registration Date'),
+}, {
+    id: 'inactive',
+    visible: false,
+    type: dataElementTypes.BOOLEAN,
+    header: i18n.t('Inactive'),
+}]
+    .map(field => ({
+        ...field,
+        isMainProperty: true,
+    }));
```

Comment:
Could this be?
```suggestion
const getDefaultMainConfig: Array<MainColumnConfig> = [{
    id: 'regUnit',
    visible: false,
    type: dataElementTypes.TEXT,
    header: i18n.t('Registering unit'),
    isMainProperty: true,
}, {
    id: 'regDate',
    visible: false,
    type: dataElementTypes.DATE,
    header: i18n.t('Registration Date'),
    isMainProperty: true,

}, {
    id: 'inactive',
    visible: false,
    type: dataElementTypes.BOOLEAN,
    header: i18n.t('Inactive'),
    isMainProperty: true,
}];

```

Reply:
Please let's leave this one as is

Reply:
I dont think this needs to be a function but I you have the last word in this

Reply:
Ah, missed the point about the function. I thought you were referring to the map at the end. Removing the function 👍

Reply:
👍